bin/configurable-http-proxy

```diff
@@ -108,12 +108,17 @@ cli
 // collects multiple flags to an object
 // --custom-header "k1:v1" --custom-header " k2 : v2 " --> {"k1":"v1","k2":"v2"}
 function collectHeadersIntoObject(value, previous) {
-  var headerParts = value.split(":").map((p) => p.trim());
-  if (headerParts.length != 2) {
-    log.error("A single colon was expected in custom header: " + value);
+  value = value.trim();
+  var colon = value.indexOf(":");
+  if (colon <= 0) {
+    log.error("A colon was expected in custom header: " + value);
```

Comment:
I think calling `log` here isn't valid. I got crashes when fidling with this part of the code to make it log using `log`.

Reply:
Good catch! Fixed and test added.